torch/quantization/fx/quantize.py

```diff
@@ -407,6 +414,696 @@ def load_arg(a):
+                        new_node, qconfig, load_arg, model, modules, observed_graph,
+                        node_name_to_target_dtype,
+                        qhandler, prepare_custom_config_dict)
+
+                    is_last_node_of_pattern = root_node is node
+                    is_like_copy_node = \
+                        (qhandler is not None and (
+                            isinstance(qhandler, CopyNodeQuantizeHandler)
+                        ))
+                    if is_last_node_of_pattern and (not is_like_copy_node):
+                        # this returns the new observer node if it was needed
+                        maybe_output_obs_node = maybe_insert_output_observer_for_node_rewrite(
+                            new_node, model, modules, observed_graph, matches,
+                            node_name_to_target_dtype, pattern, qhandler)
+                        if maybe_output_obs_node is not None:
+                            env[new_node.name] = maybe_output_obs_node
+                            env[maybe_output_obs_node.name] = maybe_output_obs_node
+
+                            # for quantized cat nodes only, we modify the graph
+                            # to make all inputs and outputs use the first input's
+                            # observer
+                            if isinstance(qhandler, CatQuantizeHandler):
+                                adjust_observers_for_cat_rewrite(
+                                    new_node, model, modules)
+
+                else:  # output
+                    prev_node = new_node.args[0]
+                    if isinstance(prev_node, Node):
+                        if is_activation_post_process_node(prev_node, modules):
+                            prev_node = prev_node.args[0]
+                    elif isinstance(prev_node, dict):
+                        # get first value
+                        prev_node = list(prev_node.items())[0][1]
+                        assert isinstance(prev_node, Node)
+                        if is_activation_post_process_node(prev_node, modules):
+                            prev_node = prev_node.args[0]
+
+                    # we check for node again because some graphs can return
+                    # None
+                    if isinstance(prev_node, Node):
+                        prev_node_qconfig = qconfig_map.get(prev_node.name, None)
+                        # this modifies new_node inplace
+                        maybe_insert_input_observers_for_node_rewrite(
+                            new_node, prev_node_qconfig, load_arg, model,
+                            modules, observed_graph,
+                            node_name_to_target_dtype,
+                            qhandler, prepare_custom_config_dict)
+            else:
+                # if we skipped observation and we have a copy node,
+                # assign the dtype to the dtype of the previous node
+                # new_node.dtype = new_node.args[0].dtype
```

Comment:
is this comment up to date? not sure what this means. is this addressed before alraedy?

Reply:
sorry, this is stale, we can remove the comment and move the code to mark 
`getitem` outputs as `torch.float` to `get_target_dtype_for_node_rewrite`